.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+How to write a good issue ? Read [this](https://github.com/containous/traefik/blob/master/.github/how-to-write-a-good-issue.md) and the [contributing guide](https://github.com/containous/traefik/blob/master/.github/CONTRIBUTING.md).
```

Comment:
Shouldn't those comments be in html comment `<!-- …` ? Otherwise it's gonna show up in the final issue if ppl don't clean it (and I guarantee that not everybody will clean those). See https://raw.githubusercontent.com/moby/moby/master/.github/ISSUE_TEMPLATE.md 👼